pkg/cmd/release/create/create.go

```diff
@@ -198,6 +202,7 @@ func createRun(opts *CreateOptions) error {
 				return fmt.Errorf("could not prompt: %w", err)
 			}
 			if tag != createNewTagOption {
+				existingTag = true
```

Comment:
Yup that is the intention. The prompt list gets populated from the remote tags so if one of those tags is selected we know it exists on the remote and then below there is no need to try to fetch the tag from the server since it does exist.